src/registrar/views/domain_request.py

```diff
@@ -493,6 +493,7 @@ def get_context_data(self):
         # Hides the requests and domains buttons in the navbar
         context_stuff["hide_requests"] = self.is_portfolio
         context_stuff["hide_domains"] = self.is_portfolio
+        context_stuff["domain_request_id"] = self.domain_request.id
```

Comment:
I know this is for pre-existing code not in scope, but thoughts on renaming this variable to `form_context` or something along the lines of that?

Reply:
I think just context also works! Glad we were in agreement 🎉